lib/turn.rb

```diff
@@ -0,0 +1,24 @@
+class Turn
+  attr_reader :guess, :card, :correct, :feedback
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if @guess == @card.answer
+      return true
```

Comment:
Ruby is really nice in that it automatically returns the value of the last line of a method so we do not need to explicitly call `return` unless we want to end a method early. We can remove the `return` here and on lines 13, 19, and 21.